.github/workflows/build-push-greenhouse-image.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}
           tags: |
             type=semver,pattern={{major}}.{{minor}}.{{patch}},value=${{ steps.read_version.outputs.IMAGE_VERSION }}
+            type=sha,enable=true,format=short,prefix=
```

Comment:
[Question]: does `prefix=` mean we don't want to give any prefix? or the value part is forgotten?

Reply:
```
prefix= --> Empty prefix means no string will be added before the SHA value.
```